images/bootstrap/Dockerfile

```diff
@@ -50,6 +50,9 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
     && apt-get clean \
     && python -m pip install --upgrade pip setuptools wheel
 
+# Cleanup unnecessary package(s)
+RUN apt-get remove -y systemd
```

Comment:
is there any reason to have this after we install things? seems like this should be before ARG IMAGE_ARG even.

Reply:
@BenTheElder if i move it any earlier the command fails ... i tried it

Reply:
?? is something depending on systemd then? what's the failure ??

Reply:
ok figured it out . needed special incantation - https://superuser.com/questions/921354/script-answer-to-apt-get-request-to-continue-type-in-the-phrase-yes-do-as-i-s